src/main/java/hu/oe/nik/szfmv/automatedcar/model/AutomatedCar.java

```diff
@@ -1,17 +1,26 @@
-package hu.oe.nik.szfmv.automatedcar;
+package hu.oe.nik.szfmv.automatedcar.model;
 
-import hu.oe.nik.szfmv.automatedcar.model.WorldObject;
+import hu.oe.nik.szfmv.automatedcar.model.interfaces.ICrashable;
+import hu.oe.nik.szfmv.automatedcar.model.interfaces.IDynamic;
 import hu.oe.nik.szfmv.automatedcar.systemcomponents.Driver;
 import hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.VirtualFunctionBus;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public class AutomatedCar extends WorldObject {
+import javax.xml.bind.annotation.XmlTransient;
 
+public class AutomatedCar extends WorldObject implements ICrashable, IDynamic {
+
+    @XmlTransient
     private static final Logger LOGGER = LogManager.getLogger();
 
+    @XmlTransient
```

Comment:
ha Car-al csináljuk akkor ha jól értelmezem ezekre nem kell majd az XmlTransient, mert majd a Car-ban annotálunk

Reply:
Akkor fog kelleni, ha az AutomatedCar-t is XML-ből olvassuk be. Megmondom őszintén nem emlékszem, hogy erről lett volna-e szó a jegyzetben, de majd átolvasom még egyszer. Ha kódból hozzuk létre, akkor természetesen nem fog kelleni.